### DISCUSSION TITLE
Pass shell executable in podman connection type

### DISCUSSION THREAD

**Discussion by ghost** [containers/ansible-podman-collections#726] [Q&A]
I am attemping to modify a container with Ansible that has an ENTRYPOINT configured that does not allow any commands. If I manually start the container with `podman run -itu root contImage /bin/bash`, I /can/ execute commands. How do you tell the podman connection type what shell to explicitly activate?

**Comment by sshnaidm** [comment]
Hi, @gpettey 
Example playbook of what you are doing would be helpful for more understanding
